.taskcluster.yml

```diff
@@ -11,7 +11,10 @@ tasks:
       $if: 'tasks_for == "github-pull-request"'
```

Comment:
Looks like you were rebasing while I was abusing master on Friday. It should be safe to rebase now.